addons/hr_timesheet/static/src/components/time_hour_field/time_hour_field.js

```diff
@@ -0,0 +1,19 @@
+import { registry } from "@web/core/registry";
+import {_t} from "@web/core/l10n/translation";
+import { FloatTimeField } from "@web/views/fields/float_time/float_time_field";
+
+export class TimeHourField extends FloatTimeField {
+    get formattedValue() {
+        const unitAmount = super.formattedValue;
+        const [hours, minutes] = unitAmount.split(":");
+        if (minutes !== '00') {
+            return _t("%(hours)sh %(minutes)sm", { hours, minutes });
```

Comment:
```suggestion
            return _t("%(hours)sh %(minutes)s", { hours, minutes });
```
I don't think the m is needed here, the spec doesn't ask for it

Reply:
![image](https://github.com/user-attachments/assets/6fb4a33e-ae50-4b1c-ae87-ca460b3484eb)
@pkyriakou  after making these changes it looks like this 
this is not expected behavior:thinking:

Reply:
it is expected behaviour - that is what the spec is calling for and iirc it's also more standard

Reply:
Thanks for the clarification! Got it now 👍